src/main/resources/mcmod.info

```diff
@@ -11,6 +11,6 @@
       "credits": "",
       "logoFile": "assets/refinedstorage/textures/logo.png",
       "screenshots": [],
-      "dependencies": []
+      "dependencies": ["inventorytweaks"]
```

Comment:
You don't need to load after InvTweaks, order of loading doesn't matter for the changes you need.